packages/api/endpoints/rules.js

```diff
@@ -55,24 +59,32 @@ async function get(req, res) {
  * @returns {Promise<Object>} the promise of express response object
  */
 async function post(req, res) {
-  const data = req.body;
-  const name = data.name;
-
-  const model = new models.Rule();
-
   try {
-    await model.get({ name });
-    return res.boom.conflict(`A record already exists for ${name}`);
-  } catch (e) {
-    if (e instanceof RecordDoesNotExist) {
-      const record = await model.create(data);
-
-      if (inTestMode()) {
-        await addToLocalES(record, indexRule);
+    const data = req.body;
+    const name = data.name;
+
+    const model = new models.Rule();
+
+    try {
+      await model.get({ name });
+      return res.boom.conflict(`A record already exists for ${name}`);
```

Comment:
We're doing `badRequest` in collections and providers. They should all be the same for consistency, whichever way you want to go.

Reply:
`conflict` is the appropriate way to go, not `badRequest`